src/api/services/adminServiceClient.ts

```diff
@@ -15,16 +21,53 @@ import {
   SharingListResponse,
 } from './adminServiceHelpers';
 
+let accessToken: string = '';
+let accessTokenExpiry: number = 0;
+
+const oktaClient = axios.create({
+  baseURL: SSP_OKTA_AUTH_SERVER_URL,
+  headers: {
+    common: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+      Authorization: `Basic ${Buffer.from(
+        `${SSP_OKTA_CLIENT_ID}:${SSP_OKTA_CLIENT_SECRET}`,
+        'binary'
+      ).toString('base64')}`,
+    },
+  },
+});
+
+async function getAccessToken() {
```

Comment:
Is there a reason not to use https://github.com/okta/okta-auth-js for this? It provides some nice features like active renewal (it'll keep the token fresh and not introduce extra latency waiting for a new one), and if we end up letting people sign in to the portal using Okta (which seems likely at some point) we would need it in the front-end anyway.

Reply:
The js SDK is for [OpenID Connect & OAuth 2.0 API](https://developer.okta.com/docs/api/resources/oidc) type workflows for signing in end users. It does not support the [Client Credentials](https://developer.okta.com/docs/guides/implement-grant-type/clientcreds/main/) workflow.

Reply:
The SDK supports the Authorization Code flow, PKCE OAuth 2.0 flow, and Implicit OAuth 2.0 flow, but not the Client Credentials flow, as it is intended for authenticating end users, not machines.